app/src/main/res/values/strings.xml

```diff
@@ -128,4 +128,7 @@
     <string name="database_error">An error occurred connecting to the database</string>
     <string name="card_icon">card icon</string>
 
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```

Comment:
I couldn't find a use of that string... If it is the case, I think we can safely delete it.

Reply:
I don't know where this come from ahaha. I'll remove it

Reply:
it is created automatically when you create a new fragment, deleting is nice :)